apps/E2E/src/ContextualMenu/specs/ContextualMenu.spec.win.ts

```diff
@@ -12,41 +12,41 @@ describe('ContextualMenu Testing Initialization', function () {
+    await expect(await ContextualMenuPageObject.didAssertPopup()).toBeFalsy(ContextualMenuPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
   });
 });
 
 describe('ContextualMenu Functional Tests', async () => {
   /* Scrolls and waits for the ContextualMenu to be visible on the Test Page */
   beforeEach(async () => {
-    await ContextualMenuPageObjectObject.scrollToTestElement();
+    await ContextualMenuPageObject.scrollToTestElement(await ContextualMenuPageObject._contextualMenu);
 
-    await ContextualMenuPageObjectObject.sendKey(ContextualMenuSelector.ContextualMenu, Keys.ESCAPE); // Reset ContextualMenu state for next test
+    await ContextualMenuPageObject.closeContextualMenu(); // Reset ContextualMenu state for next test
   });
 
-  it('Click on ContextualMenu Button and validate that the list of ContextualMenu Items open', async () => {
+  it('Click on ContextualMenu Button -> Validate that the menu opens by checking if its items are displayed', async () => {
     /* Click on the ContextualMenu */
-    await ContextualMenuPageObjectObject.clickComponent();
-    await ContextualMenuPageObjectObject.waitForContextualMenuItemsToOpen(PAGE_TIMEOUT);
+    await ContextualMenuPageObject.click(ContextualMenuPageObject._contextualMenu);
 
-    await expect(await ContextualMenuPageObjectObject.contextualMenuItemDisplayed()).toBeTruthy();
-    await expect(await ContextualMenuPageObjectObject.didAssertPopup()).toBeFalsy(ContextualMenuPageObjectObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
+    await expect(await ContextualMenuPageObject.waitForContextualMenuItemsToDisplay(PAGE_TIMEOUT)).toBeTruthy();
+
+    await expect(await ContextualMenuPageObject.didAssertPopup()).toBeFalsy(ContextualMenuPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
   });
 
-  it('Type "SPACE" to select the ContextualMenu and validate that the list of ContextualMenu Items open', async () => {
+  it("Type 'SPACE' on ContextualMenu Button -> Validate that the menu opens by checking if its items are displayed", async () => {
     /* Type a space on the ContextualMenu */
-    await ContextualMenuPageObjectObject.sendKey(ContextualMenuSelector.ContextualMenu, Keys.SPACE);
-    await ContextualMenuPageObjectObject.waitForContextualMenuItemsToOpen(PAGE_TIMEOUT);
+    await ContextualMenuPageObject.sendKeys(ContextualMenuPageObject._contextualMenu, [Keys.SPACE]);
+
+    await expect(await ContextualMenuPageObject.waitForContextualMenuItemsToDisplay(PAGE_TIMEOUT)).toBeTruthy();
 
-    await expect(await ContextualMenuPageObjectObject.contextualMenuItemDisplayed()).toBeTruthy();
-    await expect(await ContextualMenuPageObjectObject.didAssertPopup()).toBeFalsy(ContextualMenuPageObjectObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
+    await expect(await ContextualMenuPageObject.didAssertPopup()).toBeFalsy(ContextualMenuPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
   });
```

Comment:
Could we also add a test for pressing "ENTER" on Contextual Menu Button? I think both "ENTER" and "SPACE" are used to open a contextual menu.

Reply:
Feel free to disregard this comment if you're just looking to clean up/refactor existing code.

Reply:
I think the test is a small and reasonable ask. Added